src/modules/payment-app-configuration/webhook-manager.ts

```diff
@@ -0,0 +1,154 @@
+import urlJoin from "url-join";
```

Comment:
Do we need this package? We could use native `URL` and `URLSearchParams` instead